command/v7/shared/app_summary_displayer_test.go

```diff
@@ -659,10 +659,58 @@ var _ = Describe("app summary displayer", func() {
 		When("there is an active deployment", func() {
 			var LastStatusChangeTimeString = "2024-07-29T17:32:29Z"
 			var dateTimeRegexPattern = `[a-zA-Z]{3}\s\d{2}\s[a-zA-Z]{3}\s\d{2}\:\d{2}\:\d{2}\s[A-Z]{3}\s\d{4}`
+			var maxInFlightDefaultValue = 1
 
 			When("the deployment strategy is rolling", func() {
 				When("the deployment is in progress", func() {
-					When("last status change has a timestamp", func() {
+					When("last status change has a timestamp and max-in-flight is non-default", func() {
+						BeforeEach(func() {
+							summary = v7action.DetailedApplicationSummary{
+								Deployment: resources.Deployment{
+									Strategy:         constant.DeploymentStrategyRolling,
+									StatusValue:      constant.DeploymentStatusValueActive,
+									StatusReason:     constant.DeploymentStatusReasonDeploying,
+									LastStatusChange: LastStatusChangeTimeString,
+									Options: resources.DeploymentOpts{
+										MaxInFlight: 2,
+									},
+								},
+							}
+						})
+
+						It("displays the message", func() {
+							var actualOut = fmt.Sprintf("%s", testUI.Out)
+							Expect(actualOut).To(MatchRegexp(`Rolling deployment currently DEPLOYING \(since %s\)`, dateTimeRegexPattern))
+						})
+						It("displays max-in-flight value", func() {
+							Expect(testUI.Out).To(Say(`max-in-flight: 2`))
+						})
+					})
+					When("last status change has a timestamp and max-in-flight is default", func() {
+						BeforeEach(func() {
+							summary = v7action.DetailedApplicationSummary{
+								Deployment: resources.Deployment{
+									Strategy:         constant.DeploymentStrategyRolling,
+									StatusValue:      constant.DeploymentStatusValueActive,
+									StatusReason:     constant.DeploymentStatusReasonDeploying,
+									LastStatusChange: LastStatusChangeTimeString,
+									Options: resources.DeploymentOpts{
+										MaxInFlight: maxInFlightDefaultValue,
+									},
+								},
+							}
+						})
+
+						It("displays the message", func() {
+							var actualOut = fmt.Sprintf("%s", testUI.Out)
+							Expect(actualOut).To(MatchRegexp(`Rolling deployment currently DEPLOYING \(since %s\)`, dateTimeRegexPattern))
+						})
+						It("does not display max-in-flight", func() {
+							Expect(testUI.Out).NotTo(Say(`max-in-flight`))
+						})
+					})
+					// 'unset' is important for the newer-CLI-than-CAPI scenario
```

Comment:
Not 100% sure I follow what this comment is saying

Reply:
Does it make sense if I change it to:
// Scenario: an older version of CAPI that doesn't return max-in-flight

Reply:
Yeah I think it make more sense to add that information to the When block since that is going to be what is printed in case the test fail 👍

Reply:
Agreed, addressed in latest commit